lib/ui/compositing.dart

```diff
@@ -272,10 +272,10 @@ class SceneHost extends NativeFieldWrapperClass2 {
   /// to the child.
   ///
   /// The scene host takes ownership of the provided export token handle.
-  SceneHost(int export_token_handle) {
+  SceneHost(dynamic export_token_handle) {
```

Comment:
Actually, it's not so straightforward. In Mozart's lib/flutter the eventpair for the child view is created in Dart. I'll need to port some of that Dart code into C++.